docker/web-plugin-packager/README.md

```diff
@@ -0,0 +1,44 @@
+# web-plugin-packager
+
+web-plugin-packager is a docker image used to package JS plugins for the Deephaven Web UI with your own custom docker container.
+
+## Usage
+
+First, follow the instructions to [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/quickstart/#set-up-your-deephaven-deployment).
+
+Once you have the docker-compose.yml file pulled down, you must create your own web image that includes the web plugins you would like to use. Create a `Dockerfile` in the same folder as your `docker-compose.yml`, and fill it with the following:
+
+```
+# Pull the web-plugin-packager image
+FROM ghcr.io/deephaven/web-plugin-packager:latest as build
+
+# Specify the plugins you wish to use. You can specify multiple plugins separated by a space, and optionally include the version number, e.g.
+# RUN ./pack-plugins.sh <js-plugin-name>[@version] ...
+# For a list of published plugins, see https://www.npmjs.com/search?q=%22%40deephaven%2Fjs-plugin%22
+# Here is how you would install the matplotlib and table-example plugins
+RUN ./pack-plugins.sh @deephaven/js-plugin-matplotlib @deephaven/js-plugin-table-example
```

Comment:
> #8 1.953 npm ERR! 404 Not Found - GET https://registry.npmjs.org/@deephaven%2fjs-plugin-matplotlib - Not found

Reply:
I originally named it `@deephaven/matplotlib-js-plugin`, but I am going to be consistent and name them all `@deephaven/js-plugin-*`. I'll republish this one.

Reply:
I've renamed it to `@deephaven/js-plugin-matplotlib` to match these docs.